client/src/accounts/authentication/login.hook.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { describe, expect, it } from 'vitest';
+import { HTTP_BAD_REQUEST, HTTP_OK } from '~/api/api.constants';
+import { rest, server } from '~/mocks/server';
+import { act, renderHook, waitFor } from '~/test/utils';
+import { LoginForm } from './login-form.component';
+import { useLogin } from './login.hook';
+const ENDPOINT = '*/api/accounts/';
+
+interface Result {
+  isSuccess: boolean;
+  isError: boolean;
+  data: Record<string, unknown>;
```

Comment:
The shape of data returned should be known, we are verifying it in the hook, so I expect there is a type for it. Again, this is the point of types, if your not going to use them always, the code becomes a mess.